resources/assets/sass/overrides/solutions.scss

```diff
@@ -9,3 +9,18 @@
         top: 1em;
     }
 }
+
+.horizontal-scroll {
+    display: flex;
+    overflow-x: scroll;
+    overflow-y: hidden;
+}
+
+.horizontal-scroll::-webkit-scrollbar {
+  display: none; /* Hide scrollbar for Chrome, Safari and Opera */
+}
+
+.horizontal-scroll {
+  -ms-overflow-style: none; /* Hide scrollbar for IE and Edge */
+  scrollbar-width: none; /* Firefox */
+}
```

Comment:
`overflow: scroll` fără scrollbar înseamnă că:

1. nu există un visual cue că elementul este scrollable.
2. cine nu are trackpad sau mouse cu scroll orizontal nu poate accesa the overflowing content pe desktop.

Also, please keep indentation la 4 spații.